src/Sulu/Bundle/WebsiteBundle/Tests/Functional/Entity/DomainRepositoryTest.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
```

Comment:
The comment would be sufficient once :wink:

Reply:
ups there where 2 namespaces. is fixed!